scss/bitstyles/ui/Colors.stories.js

```diff
@@ -0,0 +1,76 @@
+import colorPalette from '../exports/_color-palette.scss';
+
+const ColorItem = (label, color) => {
+  return `
+    <li style="background-color: ${color};">
+      <div style="min-height:4rem;"></div>
+      <span
+        class="u-bg--white u-padding-xxs-x u-margin-xxs-left"
+        style="font-size: 10px;"
+      >
+        ${label}
+      </span>
+    </li>
+  `;
+}
+
+const ColorRow = ({ palette }) => {
+  return `
+    <ul class="u-flex__grow-1 u-grid a-list-reset" style="grid-template-columns: repeat(${
+      Object.keys(palette).length
+    }, 1fr)">
+      ${Object.keys(palette)
+        .map(color => {
+          const colorNameParts = color.split('.');
+          const colorName = colorNameParts[colorNameParts.length - 1];
+          return ColorItem(colorName, colorPalette[color]);
+        })
+        .join('')}
+    </ul>
+  `;
+};
+
+const ColorPalette = () => {
+  const values = Object.values(colorPalette);
+  const final = [];
+  let counter = 0;
+  let portion = {};
+
+  Object.keys(colorPalette).map(key => {
+    if (counter !== 0 && counter % 18 === 0) {
+      final.push(portion);
+      portion = {};
+    }
+    portion[key] = values[counter];
+    counter += 1;
+  });
+  final.push(portion);
```

Comment:
🤯

Reply:
Yeh, I suspect I would react the same in a couple of months! I’ll make it a function, and add comments to hopefully make this clearer 👍